spring-app/src/main/kotlin/pl/starchasers/up/controller/UploadController.kt

```diff
@@ -41,19 +37,16 @@ class UploadController(
      * @param file Uploaded file content
      */
     @PostMapping("/api/upload")
-    fun anonymousUpload(@RequestParam file: MultipartFile, principal: Principal?): UploadCompleteResponseDTO {
-        val user = userService.fromPrincipal(principal)
-        val contentType = ContentType(
+    fun anonymousUpload(@RequestParam file: MultipartFile): UploadCompleteResponseDTO {
+        val contentType =
             if (file.contentType == null || file.contentType!!.isBlank()) "application/octet-stream"
             else file.contentType!!
-        )
 
         return fileService.createFile(
             BufferedInputStream(file.inputStream),
-            Filename(file.originalFilename ?: "file"),
```

Comment:
i think you restored `Filename` and `Filesize` classes, but they are still removed here